ui/admin/setup-add.php

```diff
@@ -423,16 +422,19 @@
 	};
 
 	jQuery( function ( $ ) {
-		$( document ).Pods( 'validate' );
-		$( document ).Pods( 'submit' );
-		$( document ).Pods( 'wizard' );
-		$( document ).Pods( 'dependency' );
-		$( document ).Pods( 'advanced' );
-		$( document ).Pods( 'confirm' );
-		$( document ).Pods( 'sluggable' );
+		var $document = $( document );
+
+		$document.Pods( 'validate' );
+		$document.Pods( 'submit' );
+		$document.Pods( 'wizard' );
+		$document.Pods( 'dependency' );
+		$document.Pods( 'advanced' );
+		$document.Pods( 'confirm' );
+		$document.Pods( 'sluggable' );
 
+		$document.find( '.pods-dependent-toggle[data-name-clean]' ).trigger( 'change' );
 		$( '.pods-admin' ).on( 'render', '.pods-form-ui-field', function ( e ) {
```

Comment:
```suggestion

		var $admin = $( '.pods-admin' );

		$admin.on( 'render', '.pods-form-ui-field', function ( e ) {
```

Reply:
Not sure what this would fix? The `$document` relation (directly on load) is important here

Reply:
Oh wait, it's about the line below that haha, sure looks good to me!